ducktape/tests/test.py

```diff
@@ -250,6 +250,15 @@ def configure_logger(self):
         info_fh.setFormatter(formatter)
         debug_fh.setFormatter(formatter)
 
+        if self.session_context.debug:
+            # Colorize output if debugging is enabled
+            # FIXME: Might want to check for terminal capability too.
+            logging.addLevelName(logging.CRITICAL, "\033[41;33mCRIT ")
```

Comment:
Yeah, this is actually really nice, but I think a check on whether we're in a terminal is a must. These are littering my logs with escape codes. Unfortunately I don't think there's an integrated way to handle this, but an approach [like this](http://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output) might work, although I'd rather not have to do string replacement like that. Something that detects the terminal and only applies these changes (and the other one that resets the formatting) to the loggers being sent to a terminal would be much better.